apps/studio/components/interfaces/Realtime/Inspector/ChooseChannelPopover/index.tsx

```diff
@@ -44,6 +49,15 @@ export const ChooseChannelPopover = ({ config, onChangeConfig }: ChooseChannelPo
 
   const onSubmit = () => {
     setOpen(false)
+    Telemetry.sendEvent(
+      {
+        category: 'realtime_inspector',
+        action: 'started_listening_to_channel',
```

Comment:
can we make this name more obvious as to where it is? i see a very similar one here: https://github.com/supabase/supabase/pull/20672/files#diff-0a4d62883193a44ed54c99d0ae5caf753adc83af8083da5c5c87cf5e798a96bbR43.

`started_listening_to_channel` for this right? maybe rename to `started_listening_to_channel_in_input_channel_popover` or something.

<img width="340" alt="Screenshot 2024-01-23 at 13 20 23" src="https://github.com/supabase/supabase/assets/5532241/775d0e9d-70dd-4ea0-8dc1-8b952dba43eb">